### ISSUE TITLE
Plan test-launch of sixacts.org site prior to the 17th Aug at 4pm 

### ISSUE THREAD

**Issue by EvaSchlunke** [pdiveris/sixacts#141]
Can we schedule a test period (3 - 4 hours?) on the 13th August where sixacts.org is live.
This would allow us to test all functions and go through launch procedure in preparation for the actual launch on 17th Aug at 4pm.

**Comment by pdiveris** [comment]
Yes, I was thinking of mnaking the live version open for a while to be tested. But exactly how do you envisage this? Open to all? also, do you need data migrating from beta?

**Comment by EvaSchlunke** [comment]
Thank you for clarification.

No data migration necessary. Probably best we test exactly how it will go on the 17th - so that means activating it as a blank slate (no proposals)

I think we can risk making it open to all for a couple of fixed hours and I will alert individuals from participating campaign groups. Could this also be an opportunity for people to register as a user and retain these accounts on the sixacts site following the test period? (so they don't need to redo this on the 17th Aug - which will save time)

**Comment by pdiveris** [comment]
Yes, sure, agree. There is of course the option of password protecting the whole site - but that would make it innacessible to everyone else for that period. Your call.

**Comment by EvaSchlunke** [comment]
Hmmm... Lemme think about this. Will confirm on Sunday. Is that ok?
